packs/nagios/etc/st2service_handler.py

```diff
@@ -5,16 +5,24 @@
     import simplejson as json
 except ImportError:
     import json
-import requests
+import os
 import sys
-import urlparse
+
+import requests
+import yaml
 
 # ST2 configuration
-ST2_HOST = 'localhost'
-ST2_API_PORT = '9101'
+ST2_CONFIG_FILE = './config.yaml'
```

Comment:
Yeah dude. Spotted during testing and fixed in future commits. Hold your horses.